### ISSUE TITLE
NVML shared directory not found

### ISSUE THREAD

**Issue by signupdemo** [iperov/DeepFaceLab#49]
Hello,
I downloaded the prebuilt Torrent and tried to Run it. I was successfully able to create frames, but Im stuck at extracting faces.
It throws an error saying "NVML shared library not found" I have Nvidia 1060 and installed cuda 9.
I have tried Openfaceswap, it worked with half face...  but I need to model using DFaker full face. Please help.
Thanks.

**Comment by iperov** [comment]
update nvidia drivers
such library should be located here:
![explorer_2018-11-20_09-29-32](https://user-images.githubusercontent.com/8076202/48753270-d52ca800-eca6-11e8-8ddc-e79dae91827f.jpg)

**Comment by signupdemo** [comment]
Thanks for the reply, 
I downloaded the nvml and it worked, I was able to extract faces.
But now im stuck at the training step 6 onwards.  All the training methods gives me this error: 
Error: module 'keras.backend.common' has no attribute 'normalize_data_format'

**Comment by signupdemo** [comment]
Thanks for the reply...
I couldn't install the drivers from nvidia site (not compatible with this windows version error) so I downloaded the standalone nvml.dll and it worked.

However while training with DF, it shows the preview for few seconds, but then throws OOM error.
Is there a way to work it out on 1060 ? 
Im using straight out the torrent files compilation, do I need to update anything as it might be outdated ?
